source/scenes/gameplay/PPGameScene.cpp

```diff
@@ -169,7 +171,7 @@ void GameScene::update(float timestep) {
         _complete->update(timestep);
         return;
     }
-
+    float currentHealth = max((float)_state.getScoreMetric("wrongAction") - _state.getHealthBack(), 0.0f);
```

Comment:
This was addressed in a small PR i made that is open rn, but this will cause merge conflicts. To account for the timed out canvases adding to the danger bar, all we need to do is change this line to 
`max((float)(_state.getScoreMetric("wrongAction") + _state.getScoreMetric("timedOut")) - _state.getHealthBack(), 0.0f);` and it'll fix the issue of timed out canvases not affecting the danger bar. Figured we'll do it on your PR to avoid conflicts.

Reply:
Sounds good. I'll fix that now